test/utils/Test/HIE/Util.hs

```diff
@@ -1,24 +1,27 @@
 {-# LANGUAGE CPP, OverloadedStrings, NamedFieldPuns #-}
-module TestUtils
+module Test.HIE.Util
```

Comment:
Out of interest, why is this renamed?

Reply:
was looking for more consistent namespacing with the other testing tools, but really don't have a strong opinion.  easy for me to put but if you prefer the other way

Reply:
No, was just asking, dont have a preference either. Keep it :)

Reply:
done Test.Hls.Util